### PR TITLE
Split out CSS-specific instructions from self-hosting overview

### PR DESCRIPTION
**Pull request by michielbdejong**

### CODE REVIEW

**Comment by michielbdejong** [comment]
This fixes #689.

**Review by RubenVerborgh — DISMISSED** [review]
Need to hold until https://github.com/solid/test-suite/issues/132 has been decided.

**Comment by RubenVerborgh** [comment]
Closing until we have answers to the following questions:
- Do a majority of test-suite panel and editors agree that the table is a correct and representative measure of spec compatibility?
- Do a majority of test-suite panel and editors agree that the table can be compiled and published without implementer consent?

Please link to a majority vote result in order to reopen.

**Comment by timbl** [comment]
The move of infomration into more logical set of pages does *not* have to wait for test stuff to be decided, surely?

File: _posts/developers/pod-server/2019-01-01-00_overview.md [hunk 1]
```diff
@@ -22,104 +22,8 @@ The following **open-source servers** are available:
...
-docker run --rm -v ~/solid-config:/config -p 3000:3000 -it solidproject/community-server -c /config/my-config.json
-```
-
-## Accessing your server {#access}
-Open your browser and visit [http://localhost:3000/](http://localhost:3000/).
-You will be greeted by a welcome page.
-
-This welcome page is hosted on a _root Pod_,
-to which **you and your apps can read and write**.
+See [the features matrix](https://github.com/solid/test-suite#table) to find out which server implementation supports which required and/or optional parts of the Solid spec.
```
Comment (csarven) [inline _posts/developers/pod-server/2019-01-01-00_overview.md:25]: This is new information that I think is outside the scope of the issue that this PR intends to fix/resolve. I suggest to omit this for the time being.

```suggestion
```

**Review by csarven — CHANGES_REQUESTED** [review]

**Comment by csarven** [comment]
With the exception of updating as per https://github.com/solid/solidproject.org/pull/690#discussion_r864667235 , this PR is fine as far reorganising existing information goes. solidproject.org happens to include the instructions to help potential users but if that information is subject to going stale or better to link out to an authoritative document that's published by the CSS team somewhere and updated more frequently, then that can work too.

**Comment by RubenVerborgh** [comment]
> The move of infomration into more logical set of pages does _not_ have to wait for test stuff to be decided, surely?

No, but https://github.com/solid/solidproject.org/pull/690/commits/2b28f36d8bc06f53ca5f4b940bd527eebea3761e was sneaked in, as @csarven also noted. Without that commit, this PR is fine.

**Review by csarven — APPROVED** [review]

**Review by csarven — COMMENTED** [review]
I think this PR is no longer applicable as the site seems to have moved on with other changes (from other PRs): https://solidproject.org/users/get-a-pod#self-host-your-pod

@michielbdejong , can we close this?

Sorry that this PR was closed, reopened, and then gone stale. Solid Team should better process open PRs...

**Pull request closed without merging**